lib/puppet_forge/v3/base.rb

```diff
@@ -60,10 +60,15 @@ def request(resource, item = nil, params = {}, reset_connection = false, conn_op
             conn.url_prefix = "#{PuppetForge.host}"
           end
 
+          path_prefix = conn.path_prefix.clone
+          
+          unless path_prefix.end_with?("/") 
+          	 path_prefix = path_prefix + "/"
+          end
           if item.nil?
-            uri_path = "/v3/#{resource}"
+            uri_path = "#{path_prefix}v3/#{resource}"
           else
-            uri_path = "/v3/#{resource}/#{item}"
```

Comment:
It looks to me like just dropping the leading slash here will allow things to work without any other changes. `path_prefix` seems to get automatically set to `/` if the `url_prefix` doesn't include a path component.